schema.sql

```diff
@@ -59,3 +59,6 @@ CREATE TABLE visits (
     vet_id INTEGER REFERENCES vets(id) ON UPDATE CASCADE ON DELETE CASCADE,
     visit_date DATE NOT NULL
 );
+
```

Comment:
- [ ] As requested by previous reviewer, add indexes in schema.sql to reduce excution time for each query. Kindly do this if you don't mind.